ReverseTicTacToe/Logic/Game.cs

```diff
@@ -137,6 +159,10 @@ public bool AttemptMove(int i_Row, int i_Col, out eLastMoveStatus o_MoveStatus)
                 o_MoveStatus = eLastMoveStatus.Good;
                 makeMove(i_Row, i_Col);
                 moveMade = v_MoveMadeSuccessfully;
```

Comment:
when I first looked at the line "return moveMade" I didn't understand its purpose until I looked here that it is a bool for is made successfully. change the name?

Reply:
fixed